src/server/nodeinfo.ts

```diff
@@ -39,7 +39,7 @@ const nodeinfo2 = async () => {
 
 	return {
 		software: {
-			name: 'misskey',
+			name: 'groundpolis',
```

Comment:
ここはMilkeyじゃなくて大丈夫ですか？

Reply:
`package.json`だと`groundpolis-milkey`になっていますし、どうしたらいいものかと思ってますね

Reply:
nodeinfoは連合先に届くので反映しておいてもいいかもしれない

```suggestion
			name: 'groundpolis-milkey',
```

とかですかね

Reply:
そうですね、これでいいと思います